example-crates/example_rpc/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "example_rpc"
```

Comment:
please rename directory and package `example_bitcoind_rpc`. Actually it should probably be `example_bitcoind_rpc_block_polling`.